pip_audit/_format/columns.py

```diff
@@ -131,6 +137,8 @@ def _format_vuln(
         ]
         if applied_fix is not None:
             vuln_data.append(self._format_applied_fix(applied_fix))
+        if self.output_aliases:
+            vuln_data.append(",".join(vuln.aliases))
```

Comment:
Nitpick: the columnar format is meant to be human-readable and not machine-readable, so we can add a little bit of whitespace here:

```suggestion
            vuln_data.append(", ".join(vuln.aliases))
```

Reply:
Yes, this makes sense. To this effect, I've also modified the markdown format file to output in the same manner as it is (as I understand it) also intended to be human readable.